app/code/Codifi/CustomerRequest/etc/frontend/routes.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Codifi_CustomerRequest
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+ -->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:App/etc/routes.xsd">
+    <router id="standard">
+        <route id="customerrequest" frontName="customerrequest">
```

Comment:
As you set `customer/request/index` path, you must use `customer` instead of `customerrequest`